Makefile.toml

```diff
@@ -865,7 +865,7 @@ script = """
 #!/usr/bin/env bash
 
 echo "Running $(tput setaf 4)cargo clippy$(tput sgr0) checks"
-cargo clippy --all-targets --all-features
+cargo clippy --all-targets RISINGWAVE_FEATURE_FLAGS
```

Comment:
I'm hesitant about clippy's feature flag here. 

`--all-features` is convenient but will break if dynamic libs not installed correctly. (Note `--rw-static-link` and `--rw-dynamic-link` don't conflict with each other, so `--all-features` is ok if installed correctly.)

Since we don't use cargo features frequently, I think it's acceptable to miss some checks. I've checked and found the only features are `failpoints`, `sync_point`, `enable_sqlsmith_unit_test`. CI is still `--all-features`.

Reply:
> CI is still `--all-features`.

LGTM. But I'm not sure whether it'll lead to confusion for developers finding that something fails on CI but succeeds locally.